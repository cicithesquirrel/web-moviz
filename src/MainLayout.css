header {
  background-color: #222;
  padding: 20px;
  color: #fff;
  text-align: center;
  margin: 0;
}
