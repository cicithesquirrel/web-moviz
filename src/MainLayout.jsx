import React, { Component } from 'react';
import './MainLayout.css';

class MainLayout extends Component {
  render() {
    return (
      <div>
        <header>
          <h1><i className="fa fa-film"></i> Welcome to Web Moviz</h1>
        </header>
        {this.props.children}
        <footer>
          License: GPL-2.0 (Cyril Chéné, 2016)<br/>
          Icons: <a href="http://fontawesome.io/">FontAwesome</a> & <a href="http://famfamfam.com/">FamFamFam</a><br/>
          Charts: <a href="http://www.chartjs.org/">Chart.js</a>
        </footer>
      </div>
    );
  }
}

export default MainLayout;
