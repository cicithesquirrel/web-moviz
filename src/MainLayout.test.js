import React from 'react';
import renderer from 'react-test-renderer';
import MainLayout from './MainLayout';

test('MainLayout', () => {
  const component = renderer.create(
    <MainLayout />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});