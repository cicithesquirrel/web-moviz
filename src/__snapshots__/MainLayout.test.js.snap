exports[`test MainLayout 1`] = `
<div>
  <header>
    <h1>
      <i
        className="fa fa-film" />
       Welcome to Web Moviz
    </h1>
  </header>
  <footer>
    License: GPL-2.0 (Cyril Chéné, 2016)
    <br />
    Icons: 
    <a
      href="http://fontawesome.io/">
      FontAwesome
    </a>
     & 
    <a
      href="http://famfamfam.com/">
      FamFamFam
    </a>
    <br />
    Charts: 
    <a
      href="http://www.chartjs.org/">
      Chart.js
    </a>
  </footer>
</div>
`;
